examples/ui5-js-app/webapp/test/e2e/basic.test.js

```diff
@@ -61,4 +74,29 @@ describe("ui5 basic", () => {
 
         expect(await browser.asControl(searchFieldSelector).getValue()).toEqual("search Value")
     }) */
+
+    it("check binding info", async () => {
+        const title = await browser.asControl(titleSelector)
+        const bindingInfo = await title.getBinding("text")
+        const response = bindingInfo.oValue
+        expect(response).toEqual("UI5 demo")
+    })
+
+    it("check fluent api", async () => {
+        const response = await browser.asControl(buttonSelector).press().getText()
+        expect(response).toEqual("open Dialog")
+    })
+
+    it("check method chaining", async () => {
+        const newTitle = "new Title"
+
+        titleSelector.forceSelect = true
+        const title = await browser.asControl(titleSelector).setTitle(newTitle).getTitle()
+        expect(title).toEqual(newTitle)
+
+        const titleFirstTime = await browser.asControl(titleSelector).setTitle(newTitle)
+        const titleAgain = await titleFirstTime.setTitle(newTitle)
+        const response = await titleAgain.getTitle()
+        expect(response).toEqual(newTitle)
+    })
```

Comment:
could you please rephrase the "its" to better communicate that these refer to the control retrieval/value api?